app/src/main/java/org/fossasia/openevent/general/event/EventsViewModel.kt

```diff
@@ -1,5 +1,6 @@
 package org.fossasia.openevent.general.event
 
+import android.content.Context
```

Comment:
No android import in view model.

Reply:
Ill have to move the internet checking logic out of the ViewModel then. I did this because you suggested using the ViewModel in the discussions of the Issue #1341 for this PR. Should I proceed?

Reply:
Yes wrap the check in another class with injected context

Reply:
I have directly used the Util method to check for connection from EventsFragment passing the context

Reply:
The ViewModel should be pure java code, no Android imports

Reply:
@iamareebjamal sorry for the confusion. What I meant by most recent comment was referring to a new commit that I had pushed at that instant. In this, I have removed the Android import from ViewModel and have implemented checking of internet connectivity logic from the EventsFragment, passing the context from there only